packages/common/src/utils/axios.js

```diff
@@ -20,13 +20,13 @@ import Cookie from 'js-cookie';
  *    postFooBar({})
  *
  * @param {string} key The key of captcha token of the request body
- * @param {function(): Promise<string> | string} getCaptcha an sync or async function for getting captcha token
+ * @param {function(AxiosRequestConfig): Promise<string> | string} getCaptcha an sync or async function for getting captcha token
  * @return {function(AxiosRequestConfig): Promise<AxiosRequestConfig>}
  */
 export const createCaptchaInterceptor = (key, getCaptcha) => {
   return async (config) => {
     if (typeof config?.data === 'object' && key in config.data && !config.data.key) {
```

Comment:
Can we simplify the condition like below:

```
if (R.path(['data', 'key'], config)) {
```

Reply:
I'm not quite familiar with `ramda`, the logic is:
- config.data is a non-null object
- config.data has `key` (could be undefined but must exists)
- config.data[key] has no value